modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -297,6 +325,13 @@ private static Map<String, Object> _getVariables(
 			Object object = payloadJSONObject.get(
 				"model" + objectDefinition.getName());
 
+			if (oldValues) {
+				String suffix = _getSuffix(
+					objectDefinition, systemObjectDefinitionMetadataRegistry);
+
+				object = payloadJSONObject.get("original" + suffix);
```

Comment:
@p-albuquerque could you check if we can inline here?

Reply:
> Create a new variable for the right hand side operand of the '+' operator for better readability